tests/gcp/installer/util/test_data/merge_from_firewall_rules.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  - rule_id: 'prevent_allow_all_ingress'
+    description: Detect allow ingress to all policies
+    mode: blacklist
+    match_policies:
+      - direction: ingress
+        allowed: ['*']
+    verify_policies:
+      - allowed:
+        - IPProtocol: 'TESTING_ALL'
```

Comment:
What happens if the user has stripped the quotes in their rule file (suppose they cleaned it up), but our `merge_to` file has quotes?

Also vice-versa, what if something in their rule file has quotes, but our `merge_to` does not have quotes?